lms/djangoapps/course_api/blocks/tests/test_views.py

```diff
@@ -30,6 +31,14 @@ def setUpClass(cls):
         # create a toy course
         cls.course_key = ToyCourseFactory.create().id
         cls.course_usage_key = cls.store.make_course_usage_key(cls.course_key)
+        cls.discussion_id = "discussion_module_id"
+        cls.discussion = ItemFactory.create(
```

Comment:
Why is it necessary to store `discussion_id` and `discussion` as class variables? You can make `discussion_id` a local variable and remove `cls.discussion`.